backend/src/main/java/com/bakdata/conquery/models/query/preview/EntityPreviewExecution.java

```diff
@@ -186,7 +186,15 @@ private static Object renderValue(Object value, ResultType type, PrintSettings p
 		}
 
 		if (type instanceof ResultType.DateT) {
-			return CDate.toLocalDate((Integer) value);
+			// TODO this sidesteps issues with Jackson not localizing LocalDates when printing.
+			//  The proper approach would be to return LocalDates and configure Jackson/DateFormatter to request-scope localization.
+			// TODO is this actually a sneaky frontend-issue? Considering the other values are additionally rendered in the frontend?
+			return ResultType.DateT.INSTANCE.print(printSettings, value);
+		}
+
+		if (type instanceof ResultType.DateRangeT) {
+			final List<Integer> values = (List<Integer>) value;
+			return CDateRange.fromList(values);
```

Comment:
Die Lokalisierung hier ist vmtl. out-of-scope, aber dennoch ein Kommentar wert.

Momentan gibt das doch sogar nur z.B. `234/300` aus.

Reply:
ja ich mache einen kommentar dran. Ich hab es extra hochgezogen, damit sie nebeneinander sind. Die lokalisierung in Jackson zu lösen würde das vollstöndig lösen. Hier kommt  `{ min : ..., max : ....}` raus. Das FE rerendert das ganze entsprechend der locale. Ich denke ich werde da mit Kai drüber sprechen.